index.html

```diff
@@ -37,15 +37,208 @@
+        </ul>
+        <button class="project-btn flex">
+          <span>See this project</span>
+          <img
+            src="assets/images/right-arrow.svg"
+            width="20px"
+            height="15px"
+            alt=""
+            srcset=""
+          />
+        </button>
+      </article>
+      <article class="project flex">
+        <img
+          src="assets/images/project-05.svg"
+          alt="project 5 image"
+          class="project-img"
+        />
+        <p>Project name goes here</p>
+        <ul class="project-langs flex">
+          <li class="project-lang">HTML/CSS</li>
+          <li class="project-lang">Ruby on Rails</li>
+          <li class="project-lang">JavaScript</li>
+        </ul>
+        <button class="project-btn flex">
+          <span>See this project</span>
+          <img
+            src="assets/images/right-arrow.svg"
+            width="20px"
+            height="15px"
+            alt=""
+            srcset=""
+          />
+        </button>
+      </article>
+      <article class="project flex">
+        <img
+          src="assets/images/project-06.svg"
+          alt="project 6 image"
+          class="project-img"
+        />
+        <p>Project name goes here</p>
+        <ul class="project-langs flex">
+          <li class="project-lang">HTML/CSS</li>
+          <li class="project-lang">Ruby on Rails</li>
+          <li class="project-lang">JavaScript</li>
+        </ul>
+        <button class="project-btn flex">
+          <span>See this project</span>
+          <img
+            src="assets/images/right-arrow.svg"
+            width="20px"
+            height="15px"
+            alt=""
+            srcset=""
+          />
+        </button>
+      </article>
+    </section>
+
+    <!-- about me section -->
+    <section class="about flex">
+      <h3 class="title popins">About</h3>
+      <p id="about-me-details" class="popins">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitate to contact me.
+      </p>
+      <button id="get-cv-btn" class="g-btn flex popins">
+        <span>Get my CV</span
+        ><img src="assets/images/download.svg" width="24" height="24" alt="" />
+      </button>
+      <div class="details popins" id="langs">
+        <ul>
+          <h5>Languages</h5>
+          <li>JavaScript (ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+      <div class="details popins" id="frameworks">
+        <ul>
+          <h5>Frameworks</h5>
+          <li>React.js</li>
+          <li>Ruby on rails</li>
+          <li>RSpec</li>
+          <li>Capibara</li>
+        </ul>
+      </div>
+      <div class="details popins" id="skills">
+        <ul>
+          <h5>Skills</h5>
+          <li>Database Management</li>
+          <li>CLI</li>
```

Comment:
- [ ]  kindly consider all The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` according to requirement and this will make you code easy to read